res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,546 @@
+function Jockey3ME() {}
+
+// Variables
+Jockey3ME.EffectLedMeter = 0;
+Jockey3ME.EffectLedMeterValue = 1;
+Jockey3ME.LedMeterShowValue = 1;
+Jockey3ME.LedMeterShowValueTwo = false;
+Jockey3ME.scratching = [];
+Jockey3ME.jogwheelResolution = 2048;
+Jockey3ME.jogwheelSpinSpeed = 33+1/3;
+Jockey3ME.hotcueClearVal = 0;
+Jockey3ME.num_effectsValue = [0,0,0,0];
+Jockey3ME.effectsAvailable = 5; // Sets how many Effects are Loadable
+Jockey3ME.fxSelectKnobPush = [0,0,0,0];
+Jockey3ME.fxSelectKnobPushIterator = [0,0,0,0,0,0,0,0,0,0,0,0];
+Jockey3ME.fxSelectKnobPushLedTemp = 0;
+Jockey3ME.fxSelectKnobParamChose = 0;
+Jockey3ME.fxSelectKnobParamLinkChose = 0;
+Jockey3ME.fxSelectKnobParamLinkInverseChose = 0;
+Jockey3ME.loop_move_value = 4; // Sets how many a Loop Jumping Beats when "MOVE" is Turned
+Jockey3ME.loop_move_bool = false;
+Jockey3ME.MixerDeck1 = 0;
+Jockey3ME.MixerDeck2 = 0;
+
+script.crossfaderCurve = function (value, min, max) {
```

Comment:
If this stays just in this script, I think it would make more sense to move this code into the Jockey3ME.crossfaderCurve function (line 445) rather than overwriting script.crossfaderCurve